packages/strapi-admin/models/Role.settings.json

```diff
@@ -0,0 +1,21 @@
+{
+  "collectionName": "strapi_role",
```

Comment:
I'm not entirely sold on this collection name, is there a standard that we should be applying here?

Reply:
As @alexandrebodin replied in the other thread, this is the first iteration that we need to begin the development of other related features.
This name will most likely be changed in the future.